pages/pizza.js

```diff
@@ -200,7 +200,7 @@ import {
               </span>
             </Heading>
             <Text variant="lead">
-              GitHub is providing pizza grants to 850+ Hack Clubs.{' '}
+              GitHub is providing pizza grants to <strong>thousands</strong> Hack Clubs.{' '}
```

Comment:
To what Felix said ^

```suggestion
              GitHub is providing pizza grants to <strong>thousands</strong> of Hack Clubs.{' '}
```